build-common.xml

```diff
@@ -1901,7 +1901,6 @@ rerun your task.
 						test.type="@{test.type}"
 					>
 						<misc>
-							<jvmarg value="-Dliferay.lib.portal.dir=${project.dir}/lib/portal" />
```

Comment:
@dantewang Should we replace this with shielded container dir, this seems to be used by unit test.

Reply:
@tinatian I think this can be removed as long as no relevant unit test fails. I don't see unit test depending on the value of the portal lib dir but I'll double check.

Reply:
@dantewang Aha, I see your point. Then let's wait for ci test results